cros_gralloc/cros_gralloc_driver.cc

```diff
@@ -35,6 +44,94 @@ cros_gralloc_driver::~cros_gralloc_driver()
 		drv_render_ = nullptr;
 		close(fd);
 	}
+
+	if (drv_ivshmem_) {
+		int fd = drv_get_fd(drv_ivshmem_);
+		drv_destroy(drv_ivshmem_);
+		drv_ivshmem_ = nullptr;
+		close(fd);
+	}
+	//pthread_join(tid, NULL);
```

Comment:
if child thread is still running while we exit main thread, it should lead to some problem. It is better we check if child thread is still running, if yes, need call pthread_cancel to exit it.